hsds/link_dn.py

```diff
@@ -136,86 +136,146 @@ async def GET_Links(request):
     return resp
 
 
-async def GET_Link(request):
-    """HTTP GET method to return JSON for a link"""
+async def POST_Links(request):
+    """HTTP POST method to return JSON for a link a given set of links """
```

Comment:
`for a link a given set of links` -> `for a link or a given set of links`

Reply:
Fixed